AgileVentures/sfn-client
# Replace Redux Promise with Redux Thunk

#### How should this be manually tested?
Start up the back-end, following instructions on the [Sing for Needs project page](https://github.com/AgileVentures/sing_for_needs/)
1. On terminal, run command ...
yarn start
2. Add {console.log(this.props.artists)} to the render method in Artists.js
3. Go to the Artists page
4. Check that the array of 100 artists is logged to the console

#### What are the relevant Github Issues ?
Fixes #335
